aspnetcore/client-side/bundling-and-minification.md

```diff
@@ -131,6 +190,29 @@ This environment tag will render the bundled and minified CSS files only when ru
 
 [!code-html[Main](../client-side/bundling-and-minification/samples/WebApplication1/src/WebApplication1/Views/Shared/_Layout.cshtml?highlight=5&range=12-17)]
 
+## Consuming bundleconfig.json from Gulp
+
+If your app bundling and minification workflow requires additional processes such as image processing, cache busting or CDN assest processing etc., then you can convert the Bundle and Minify process to Gulp.
+
+> [!NOTE]
+> Conversion option only available in Visual Studio 2015 or 2017.
```

Comment:
the option is available in 2015 as well and the extension in the marketplace states 2015, 2017.  However, changing for overall support messaging.